core/templates/dev/head/css/oppia.css

```diff
@@ -839,6 +867,8 @@ textarea {
 
 .navbar-default .navbar-container {
   background: #009688;
+
```

Comment:
Sorry, is that the spaces between the initial skiptocontent block and the active/focus/hover block? I'll have the blank spaces in navbar-default removed. Not too sure how those random spaces occurred as I didn't touch the code that far down...